mod/dfrn_confirm.php

```diff
@@ -537,7 +537,7 @@ function dfrn_confirm_post(App $a, $handsfree = null)
 		if (DBA::isResult($contact)) {
 			$photo = $contact['photo'];
 		} else {
-			$photo = System::baseUrl() . '/images/person-175.jpg';
+			$photo = System::baseUrl() . '/images/person-300.jpg';
```

Comment:
There is no "person-300.jpg" on our system.

Reply:
The PR actually creates one.

Reply:
I added it.